Documentation/project-docs/performance-tests.md

```diff
@@ -103,3 +105,6 @@ Test cases should adhere to the following guidelines, within reason:
   * Pass intermediate values to a volatile static field. If the value is a struct, compute a value dependent on the structure, and store that in a volatile static field.
```

Comment:
nit: may want to mention here that you've done this in the preceding example (same as you mentioned that regarding InnerIterationCount).